src/diffusers/schedulers/scheduling_heun_discrete.py

```diff
@@ -224,9 +224,10 @@ def scale_model_input(
 
     def set_timesteps(
         self,
-        num_inference_steps: int,
+        num_inference_steps: int = None,
```

Comment:
```suggestion
        num_inference_steps: Optional[int] = None,
```

Also, adjust docstring.